android-components/.buildconfig.yml

```diff
@@ -4,150 +4,257 @@ projects:
     path: components/browser/domains
     publish: true
     upstream_dependencies:
```

Comment:
This now appears to be full dependencies? If so, `upstream_dependencies` ought to be renamed. (I know that technically they're all still upstream dependencies...but we usually mean "direct upstream" when we say such things.)

Reply:
Oh TIL! I'd always assumed "upstream dependencies" included both direct and transitive ones

Reply:
Now I'm doubting myself. I will double check this assumption later...

Reply:
OK, this is correct on `main` at the moment. For example:
- `browser-engine-gecko` depends on `concept-engine`
- `concept-engine` currently depends on `support-base`, which `browser-engine-gecko` does not explicitly depend on

In any case -- feel free to keep `upstream_dependencies` if you want. I was mainly pointing out that in a lot of our usage in other places (and currently here), that it usually means direct upstream only (regardless of whether or not that's technically correct).